src/main.js

```diff
@@ -1,5 +1,15 @@
-function main() {
-  console.log('main의 내용을 채워주세요');
+import CarRacingGame from './app/CarRacingGame.js';
+import readLineAsync from './utils/readLineAsync.js';
+
+async function main() {
+  const carRacingGame = new CarRacingGame();
+
+  const name = await readLineAsync(
+    CarRacingGame.CONSOLE_MESSAGE.PROMPT_CAR_NAMES,
+  );
+  const players = name.split(',');
+
+  carRacingGame.register(players).play(5).printWinners();
```

Comment:
```suggestion
  carRacingGame.register(name).play(5).printWinners();
```

객체지향에는 "다형성" 이라는 개념이 존재해요! 
사실 객체지향에만 쓰이는 개념은 아닌 것 같은데..

https://www.tcpschool.com/java/java_usingMethod_overloading

가령, 이런 모습인거죠. 인자의 타입이 무엇이 들어오든 정상적으로 처리하게 됩니다.

```js
Number(1)
Number("1");
Number(true);
Number(undefined);
Number("a")
```

어쨋든 Number는 숫자형을 반환하고 있어요.

마찬가지로 레이싱 게임도 배열 형태의 값을 받아올 수도 있고, 문자열을 받아올 수도 있도록 만들어주면 어떨까 싶어요!

Reply:
타입을 마구 제한한다고 좋은 건 아니겠군요...!